tesseract_urdf/test/tesseract_urdf_mesh_unit.cpp

```diff
@@ -23,12 +23,19 @@ static std::string getTempPkgPath()
 TEST(TesseractURDFUnit, parse_mesh)  // NOLINT
 {
   tesseract_common::GeneralResourceLocator resource_locator;
+  const bool global_make_convex = false;
```

Comment:
@Levi-Armstrong how do you want to handle unit tests for the `global_make_convex = true` case? I don't really want to copy-paste all the unit tests, but I don't know how many unit tests are worth making with the inverse case

Reply:
@marip8 Is this still relevant?

Reply:
The previous unit tests check the parsing of meshes both with and without the conversion to convex hulls. This unit test just checks the workflow for parsing a mesh without converting to convex hull.

I think I could re-introduce the tests from the convex mesh unit test (that I deleted) and turn those into the tests that check that `global_make_convex = true` case

Reply:
Yea, it is probably good idea to keep and update the tests you deleted.